tests/framework/session/conftest.py

```diff
@@ -466,9 +467,15 @@ def mock_import(mocker):
     # KedroSession eagerly validates that a project's settings.py is correct by
     # importing it. settings.py does not actually exists as part of this test suite,
     # so its import is mocked.
-    settings_module = f"{MOCK_PACKAGE_NAME}.settings"
+    settings_module_name = f"{MOCK_PACKAGE_NAME}.settings"
+    settings_module_location = Path(f"src/{MOCK_PACKAGE_NAME}/settings.py").resolve()
```

Comment:
There's probably no need to resolve path, I just tried to make it as "realistic" as possible (since I think modules usually have absolute paths); happy to go either way on this.

Reply:
```suggestion
    settings_module_location = f"src/{MOCK_PACKAGE_NAME}/settings.py"
```
Maybe we shouldn't resolve...